ocs_ci/ocs/resources/ocs.py

```diff
@@ -191,20 +196,26 @@ def ocs_install_verification(timeout=0):
     ocs_csv = CSV(
         resource_name=ocs_csv_name, namespace=namespace
     )
-    assert ocs_csv.check_phase(phase="Succeeded"), (
-        "OCS CSV is not in Succeeded phase!"
-    )
+    log.info(f"Check if OCS operator: {ocs_csv_name} is in Succeeded phase.")
+    ocs_csv.wait_for_phase(phase="Succeeded", timeout=timeout)
 
     # Verify OCS Cluster Service (ocs-storagecluster) is Ready
     log.info("Verifying OCS Cluster service")
-    storage_cluster = OCP(kind='StorageCluster', namespace=namespace)
-    storage_clusters = storage_cluster.get()
-    for item in storage_clusters['items']:
-        name = item['metadata']['name']
-        log.info("Checking status of %s", name)
-        assert item['status']['phase'] == 'Ready', (
-            f"StorageCluster {name} not 'Ready'"
+    storage_clusters = StorageCluster(namespace=namespace)
+    for item in storage_clusters.get()['items']:
+
+        storage_cluster_name = item['metadata']['name']
+        storage_cluster = StorageCluster(
+            resource_name=storage_cluster_name, namespace=namespace
+        )
+        log.info("Checking status of %s", storage_cluster_name)
+        log.info(
+            f"Check if StorageCluster: {local_storage_name} is in"
+            f"Succeeded phase"
         )
+        storage_cluster.wait_for_phase(
+            phase='Ready', timeout=timeout
+        ), f"StorageCluster {storage_cluster_name} not 'Ready'"
```

Comment:
I'm not sure, if I miss something, but what the string `f"StorageCluster {storage_cluster_name} not 'Ready'"` means here? Isn't it leftover from the previous `assert`?

Reply:
Yep, thanks for catch but hopefully it didn't cause any issue as I saw that verification passed. DONE